### DISCUSSION TITLE
what is the rollup?

### DISCUSSION THREAD

**Discussion by leo-brown244** [Cyfrin/foundry-full-course-cu#1980] [Q&A]
I want to know more detail.

and then, also about optimize rollup~

**Comment by EngrPips** [comment]
Hello, @leo-brown244, This article `https://www.coindesk.com/learn/what-are-rollups-zk-rollups-and-optimistic-rollups-explained/` seems like a good start.

**Reply by leo-brown244** [reply]
thanks a million. The article makes me to understand the rollup and both type of rollup concepts.

**Reply by Emperor10001** [reply]
You are welcome, Friend. I am glad that helps.
